.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+test.md
```

Comment:
- [ ] You did a great job 😃 by adding the test file to gitignore, but I see that you put it in the gitignore after pushing the files to Github, so please 😊 try to put the test firstly in the gitignore after that push again to make the test file not existing in the repo repository as stated by the previous reviewer 😃